test/inference/gibbs_conditional.jl

```diff
@@ -0,0 +1,121 @@
+using Random, Turing, Test
+using Clustering
+
+dir = splitdir(splitdir(pathof(Turing))[1])[1]
+include(dir*"/test/test_utils/AllUtils.jl")
+
+
+@turing_testset "gibbs conditionals" begin
+    Random.seed!(100)
+    
+    @turing_testset "gdemo" begin
+        N = 1000
+        (α_0, θ_0) = (2.0, inv(3.0))
+        λ_true = rand(Gamma(α_0, θ_0))
+        σ_true = sqrt(1 / λ_true)
+        m_true = rand(Normal(0, σ_true))
+        x = rand(Normal(m_true, σ_true), N)
+
+        # The conditionals and posterior can be formulated in terms of the following statistics:
+        x_bar = mean(x) # sample mean
+        s2 = var(x; mean=x_bar, corrected=false) # sample variance
+        m_n = N * x_bar / (N + 1)
+        
+        @model function inverse_gdemo(x)
```

Comment:
Is there a reason for not using the `gdemo` model directly (https://github.com/TuringLang/Turing.jl/blob/99ba1590dfbb121a38db55c8d9b5da8f2d4e22d4/test/test_utils/models.jl#L10-L16 - the main difference seems to be the number of observations?) or using an InverseGamma prior for the variance?

Reply:
Yes: because I found it easier to express (or primarily, derive) the conditional in the precision parametrization.  But if someone cares to write conditionals for `gdemo`, I don't mind.

Reply:
Aren't they basically the same, just using and return the inverse?